CONTRIBUTING.md

```diff
@@ -273,6 +273,17 @@ Thus we ask Java contributions to follow these simple guidelines:
 - `{` on same line as method name
 - in all other aspects, follow the [Oracle Java Style Guide](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 
+### Preferred ways to use timeouts in tests
+
+There is a number of ways timeouts can be used in Akka tests. If you're sure a spec is really fast, a default of `300.millis` (this DSL is obtained from importing `scala.concurrent.duration._`) is ok, however very often this is not enough as Jenkins server may GC heavily causing spurious errorstest failures, thus the following ways to use timeouts are recommended (in order of preference):
```

Comment:
That is 99% of the time not ok and no reason for it. I don't know why everyone tries to use such small timeouts. A longer timeout will not slow down the test, aside from when used in `expectNoMsg` or other timing sensitive testing. This should not be the first paragraph.